engine/kv_engine.cpp

```diff
@@ -428,8 +426,7 @@ Status KVEngine::RestoreSkiplistHead(DLRecord *pmem_record, const DataEntry &) {
 Status KVEngine::RestoreStringRecord(StringRecord *pmem_record,
                                      const DataEntry &cached_entry) {
   assert(pmem_record->entry.meta.type & StringRecordType);
-  StringView pmem_key = pmem_record->Key();
-  std::string key(string_view_2_string(pmem_key));
+  std::string key(string_view_2_string(pmem_record->Key()));
```

Comment:
string_view.hpp offers explicit conversion operator from string_view to string.
`std::string key{pmem_record->Key};` is okay now.